lib/ts/recipe/emailpassword/emaildelivery/services/index.ts

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+export { default as STMP } from "./smtp";
```

Comment:
This mean the user will do:
```
import STMP from "supertokens-node/recipe/emailpassword/emaildelivery/services";

emailDelivery: SMTP({...});
```

I think it should be `getSMTPService` instead.

Reply:
You also need to expose this file via the root of the lib. Else they will have to import from `/lib/build/..`

Reply:
And all styles of importing should work:
- `import * as ..`
- `import ABC from ...`
- `import { ABC } from ...`

Just like how everything is exported

Reply:
Can you confirm this is the case?